datarobot_batch_scoring/batch_scoring.py

```diff
@@ -677,6 +679,23 @@ def batch_generator_args(self):
         return args
 
 
+def is_redirect(req, ui):
+    """
+    test whether a request was redirect.
+    Log a warning to the user if it was redirected
+    """
+    history = req.history
+    if history:
+        first = history[0]
+        if first.is_redirect:
+            starting_endpoint = first.url  # Requested url
+            redirect_endpoint = first.headers.get('Location')  # redirect
+            if str(starting_endpoint) != str(redirect_endpoint):
+                ui.warning('The requested URL:\n\t{}\n\twas redirected '
```

Comment:
I didn't include tests because this function just logs. It would be a lot of work to simulate this in our fixtures and I don't know if it would have many benefits.